airqo-platform/AirQo-api
# Incentives setup

- What does this PR do?
This sets up the incentives microservice that utilizes the MTN momo api. Implemented the transfer  endpoint.

- What JIRA task is related to this PR?
link to the JIRA board task. Or card number. 

- How do I test out this PR?
Git clone this repo
`cd AirQo-api/src/device-registry`
Then run `npm install `to get all the necessary dependencies
Afterwards run `npm run test` to test out the end points using any REST client tool available to you. These tools include Postman or Advanced REST client on Chrome.

- Which end points should are ready for testing?
partial url:  /api/v1/incentives/transfer
